### PR TITLE
New pixel type with timestamps (MuPixel) 

### PR DESCRIPTION
**Pull request by dvbruch**
@tbisanz The new pixel type lookup works, I checked it on a sample run.

### CODE REVIEW

**Comment by tbisanz** [comment]
Hey, I'm aware of this, but I still need to figure out how we smoothly migrate to C++11 (and we totally will!) This might take a day or two but I'll merge ASAP.

File: src/EUTelProcessorGeometricClustering.cc [hunk 1]
```diff
@@ -327,184 +328,177 @@ void EUTelProcessorGeometricClustering::geometricClustering(LCEvent * evt, LCCol
 		minX = minY = maxX = maxY = 0;
 		geoDescr->getPixelIndexRange( minX, maxX, minY, maxY );
```
Comment (tbisanz) [inline src/EUTelProcessorGeometricClustering.cc]: I guess most of this is just indentation? If possible don't change this as this makes comparing versions a pain in the ass!

File: src/EUTelAPIXTbTrackTuple.cc [hunk 2]
```diff
@@ -54,6 +54,8 @@ EUTelAPIXTbTrackTuple::EUTelAPIXTbTrackTuple()
   p_tot(NULL),
   p_iden(NULL),
   p_lv1(NULL),
+  p_hitTime(NULL),
+  p_frameTime(NULL),
   _euhits(NULL),
   _nHits(0),
   _hitXPos(NULL),
```
Comment (tbisanz) [inline src/EUTelAPIXTbTrackTuple.cc]: Can you increase the version number?

Comment (dvbruch) [inline src/EUTelAPIXTbTrackTuple.cc]: You always increase the version number whenever there is a change to this file?

Comment (tbisanz) [inline src/EUTelAPIXTbTrackTuple.cc]: Yes

**Comment by tbisanz** [comment]
Just to let you know - the nightly test cases still work as before *yay* - although we get way more warnings then before (mostly deprecated auto_ptr I'd guess). This means we more or less successfully switched to C++11 (as of gcc 8.2, corresponding to devtoolset-2). Please use new C++11 feautures supported by gcc 4.8 as much as possible!

**Pull request merged**